UVAtlasTool/Mesh.cpp

```diff
@@ -1087,6 +1089,59 @@ HRESULT Mesh::GetVertexBuffer(_Inout_ DirectX::VBWriter& writer) const
     return S_OK;
 }
 
+//--------------------------------------------------------------------------------------
+void Mesh::ExportToOBJ(const wchar_t* szFileName) const
```

Comment:
Move these methods to ``MeshOBJ.cpp`` which is where I have the WaveFront OBJ reader method.

Reply:
done